### PR TITLE
Update spanish translation.

### PR DESCRIPTION
**Pull request by w3cj**

### CODE REVIEW

**Comment by w3cj** [comment]
@angelhdz I created a pull request for you. 😉 

@AndCotOli Can you look over Angel's updates?

File: README_SPANISH.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 **Lee el mensaje entero, o no podrás tener acceso al resto de los canales del servidor.**
 
-Hola amigo! Bienvenido al servidor de Discord de Coding Garden.
+¡Hola amigo(a)! Bienvenido(a) al servidor de Discord de Coding Garden.
```
Comment (w3cj) [inline README_SPANISH.md]: Nice! I want this to be gender neutral if possible.

**Comment by AndCotOli** [comment]
Sure

File: README_SPANISH.md (hunk 1 above)
Comment (AndCotOli) [inline README_SPANISH.md]: Gender neutrality is better expressed with /a -> _amigo/a_

File: README_SPANISH.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
-Antes de que puedas acceder al resto de los canales, debes presentarte en #🗣introductions (mínimo 40 carácteres) y aceptar el código de conducta reaccionando con un emoticono :point_down:.
+Para tener acceso al resto de los canales, debes presentarte en #🗣introductions (mínimo 40 caracteres) y aceptar el código de conducta reaccionando con un emoticono :point_down:.
```
Comment (AndCotOli) [inline README_SPANISH.md:5]: Typo: _carácteres_

File: README_SPANISH.md [hunk 3]
```diff
@@ -26,16 +26,16 @@ Haz una donación: <https://streamlabs.com/CodingGardenWithCJ>
...
 
 **Código de Conducta**
 
-Al registrate y enviar mensajes en este servidor, aceptas acatar el código de conducta. Cualquier violación a este puede resultar en un baneo temporal de los moderadores (pruners). Se amable!
+Al registrate y enviar mensajes en este servidor, aceptas cumplir con el código de conducta. Cualquier violación a este puede resultar en un baneo temporal de los moderadores (pruners). ¡Sé amable!
 
-* **Usa un lenguaje apropiado.** Cualquiera de estos comportamiento es inaceptable:
-  * Comentarios ofensivos hacía la identidad y expresión de género, orientación sexual, raza, etnia, idioma, tamaño, abilidad, clase, religión, cultura, subcultura, opinión política, edad, nivel de habilidades, profesión o antecedentes. 
+* **Usa un lenguaje apropiado.** Cualquiera de estos comportamientos es inaceptable:
+  * Comentarios ofensivos hacía la identidad y expresión de género, orientación sexual, raza, etnia, idioma, tamaño, neurotipo, habilidad, clase, religión, cultura, subcultura, opinión política, edad, nivel de habilidades, profesión o antecedentes.
```
Comment (AndCotOli) [inline README_SPANISH.md:38]: _Neurotipo_ doesn't exist in Spanish.

**Review by AndCotOli — CHANGES_REQUESTED** [review]
Just 3 little changes, good job @angelhdz !

**Comment by AndCotOli** [comment]
And thanks for fixing my typos!

**Comment by angelhdzmultimedia** [comment]
Good! Team effort!

About the neutotype thingy... you're right, but I think it doesn't exist yet because it's a concept that it's not very talked in Spanish cultures....

I was researching before adding that, and while the term neurotype doesn't exist in Spanish, there are topics about Neurotic conditions and Spectrum conditions. I didn't even knew about the neurotype term in English, I learned it from The Coding Train, and from the Netflix show Atypical.

So I guess, the term will eventually make it to the Spanish slang. But not currently. There isn't an equivalent I believe.

Eagerly to read your feedback. Happy Holidays!

**Comment by w3cj** [comment]
@angelhdz If you push a commit to your fork / branch (or [edit on github](https://github.com/angelhdz/discord-welcome-code-of-conduct/tree/patch-1)) with the changes suggested, it will show up here.

Google translate for neurotype translates it to neurotipo: https://translate.google.com/#view=home&op=translate&sl=en&tl=es&text=neurotype

I actually really like the show Atypical! Do they have a spanish overdub?

**Comment by angelhdzmultimedia** [comment]
@w3cj it's on Netflix.  According to my Google search, Atypical has Spanish audio and subtitles.

File: README_SPANISH.md (hunk 2 above)
Comment (angelhdzmultimedia) [reply README_SPANISH.md:5]: http://lema.rae.es/dpd/srv/search?key=car%E1cter

**Pull request merged**